.gitignore

```diff
@@ -12,8 +12,9 @@ bench_output.txt
 sample.log
 scripts/package-lock.json
 target
+# Generated by `make check-component-features`
+target-*/
```

Comment:
Why ignore `target-*`, but delete `target-feature-*` below?

Wouldn't it be better if they were both `target-feature-*`?

Reply:
Yeah, changed it on the Make side and forgot to change here.